src/scss/includes/map_theme.scss

```diff
@@ -116,16 +129,67 @@
 
 @media(max-width: 640px) {
   .map_control__scale_attribute_container {
-    .mapboxgl-ctrl-attrib {
+    background: none;
+    position: fixed;
+    left: 0px;
+    bottom: 4px;
+    width: auto;
+    z-index: 1;
+
+    .mapboxgl-ctrl.map_control__scale {
+      background: none;
+      float: right;
+      margin: 4px 5px 2px 45px;
+      line-height: 10px;
+      height: 13px;
+      position: relative;
+      border: none;
+
+      &.fadeOut {
+        transition: opacity 2s;
+        opacity: 0;
+      }
+
+      &:before {
+        content: '';
+        background: rgba(255, 255, 255, 0.9);
+        border-radius: 10px;
+        height: 20px;
+        width: calc(100% + 20px);
+        left: -8px;
+        top: -3px;
+        z-index: -1;
+        position: absolute;
+      }
+
+      &:after {
```

Comment:
This is a lot of CSS to get this looks. I see why the `:before` pseudo may be useful, but is there something which prevents keeping the same structure as the desktop control, using a border directly on the element instead of an `:after`?

Reply:
sadly no, I didn't find any other solution than this :after trick to keep the scale element with black straight borders, and have a rounded background behind it. On desktop, the rounded background is applied to the parent of the scale and the attribution, but that's not possible here, as I want to make the scale disappear while the (i) button stays visible.